voicevox_engine/app/routers/tts_pipeline.py

```diff
@@ -345,6 +345,23 @@ def sing_frame_audio_query(
             outputStereo=False,
         )
 
+    @router.post(
+        "/sing_frame_volume",
+        response_model=list[float],
+        tags=["クエリ編集"],
+        summary="スコア・歌唱音声合成用のクエリから、新しいフレーム毎音量を得る",
```

Comment:
API良さそうに思いました！！
ここの説明だけ周りと合わせる感じでこんな感じが良さそう！
（`python -m pytest --snapshot-update`でE2E更新が必要）
```suggestion
        summary="スコア・歌唱音声合成用のクエリからフレームごとの音量を得る",
```